flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvFormatStatisticsReportTest.java

```diff
@@ -61,7 +62,7 @@ protected String[] properties() {
     }
 
     @Test
-    public void testCsvFormatStatsReportWithSingleFile() throws Exception {
+    void testCsvFormatStatsReportWithSingleFile() throws Exception {
         // insert data and get statistics.
         DataType dataType = tEnv.from("sourceTable").getResolvedSchema().toPhysicalRowDataType();
         tEnv.fromValues(dataType, getData()).executeInsert("sourceTable").await();
```

Comment:
this is not related to your changes however it relates to assertj.
In the next line
```java
assertThat(folder.listFiles()).isNotNull().hasSize(1);
```
`isNotNull()` is redundant check since same check is done inside `hasSize`

Also it's better to replace `assert files != null;` with assertj's version